otherlibs/stdune/src/io.ml

```diff
@@ -336,6 +336,20 @@ let portable_symlink ~src ~dst =
         Unix.symlink src dst)
     | exception Unix.Unix_error _ -> Unix.symlink src dst
 
+let copy_atomically f ?chmod ~src ~dst () =
```

Comment:
why does this function need to take `f`? It looks like it's only ever called with `copy_file`

Reply:
Fixed. Good catch.